public/js/utils/source-map.js

```diff
@@ -15,7 +15,11 @@ function restartWorker() {
     getValue("baseWorkerURL") + "source-map-worker.js"
   );
 }
-restartWorker();
+
+// Only start the worker if sourcemaps are enabled.
+if(isEnabled("sourceMaps")) {
+  restartWorker();
+}
```

Comment:
hmm, i think it might be better to add a `stopWorker` method which we can call when we end the tests. We will likely work on source maps next sprint and need this then.

```diff
diff --git a/public/js/utils/source-map.js b/public/js/utils/source-map.js
index cf3760f..a6b31d7 100644
--- a/public/js/utils/source-map.js
+++ b/public/js/utils/source-map.js
@@ -8,13 +8,18 @@ const { isEnabled, getValue } = require("../feature");

 let sourceMapWorker;
 function restartWorker() {
-  if (sourceMapWorker) {
-    sourceMapWorker.terminate();
-  }
+  stopWorker();
   sourceMapWorker = new Worker(
     getValue("baseWorkerURL") + "source-map-worker.js"
   );
 }
+
+function stopWorker() {
+  if (sourceMapWorker) {
+    sourceMapWorker.terminate();
+  }
+}
+
 restartWorker();

 const sourceMapTask = function(method) {
@@ -148,5 +153,6 @@ module.exports = {
   getGeneratedSourceId,
   createSourceMap,
   clearData,
-  restartWorker
+  restartWorker,
+  stopWorker
 };
```